data/accountManager.go

```diff
@@ -129,7 +140,7 @@ func (manager *AccountManager) AddParticipation(participation account.PersistedP
 		return false
 	}
 
-	manager.partKeys[partkeyID] = participation
+	manager.partKeys[partkeyID] = accountPartKey{participation, ephemeral}
```

Comment:
if a key is ephemeral, I imagined you could just skip adding it to the partKeys map. It won't exist after this function returns so it doesn't need to be remembered.

Reply:
Updated...let me know what you think.